procedural/examples/foo-procedures/src/main/java/gov/nasa/ammos/aerie/procedural/examples/fooprocedures/procedures/SampleProcedure.java

```diff
@@ -0,0 +1,32 @@
+package gov.nasa.ammos.aerie.procedural.examples.fooprocedures.procedures;
+
+import gov.nasa.ammos.aerie.procedural.scheduling.plan.EditablePlan;
+import gov.nasa.ammos.aerie.procedural.scheduling.Goal;
+import gov.nasa.ammos.aerie.procedural.scheduling.annotations.SchedulingProcedure;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.ammos.aerie.procedural.timeline.payloads.activities.DirectiveStart;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+@SchedulingProcedure
+public record SampleProcedure(int quantity) implements Goal {
+  @Override
+  public void run(@NotNull final EditablePlan plan) {
+    final var firstTime = Duration.hours(24);
+    final var step = Duration.hours(6);
+
+    var currentTime = firstTime;
+    for (var i = 0; i < quantity; i++) {
+      plan.create(
+          "BiteBanana",
+          new DirectiveStart.Absolute(currentTime),
+          Map.of()
+      );
+      currentTime = currentTime.plus(step);
+    }
+    plan.commit();
+//    var results = plan.simulate(new SimulateOptions());
+//    var size = results.instances().collect().size();
```

Comment:
This is an intended example showcasing sim control

Reply:
That is not indicated in the file. I'd also recommend extracting this example into a new, proper sample procedure to demonstrate this functionality rather than leaving it as two commented-out lines of code without even an explanation comment. Extracting it into an example will also force us to _have_ an example of a procedure with sim control that we can use in tests.

I'll also say that if this example is intended to be referenced directly by users as a sort of "baby's first scheduling procedure", the procedure should be properly commented, probably a bit more redundantly than we usually would (since the user may not have the context or Java background to understand it intuitively).

Reply:
Yep we have plans for a more formal procedural tutorial down the line. This is just a sample procedure used for testing, not a "baby's first scheduling procedure". The commented code was more for developers to easily test with and without sim results. I can remove it if you'd like

Reply:
I mean, I would prefer it put into another sample procedure so can be used in tests, similar to how we have multiple sample activity types for the various simple behaviors directives can do (ie decomposing, checking daemon tasks, changing resources, validating activity duration types).

Reply:
We have `StayWellFed` and `SimluationDemo` as well, but they aren't hooked up to tests.

Reply:
If we have demo procedures for sim control, then yes, please remove the lines